e2e/cypress/integration/channel_settings/channel_header_spec.js

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {getRandomId} from '../../utils';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
```

Comment:
don't we usually have some statement about our convention using # * and so?

Reply:
✅  Updated as per request/discussion